docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -4290,16 +4290,26 @@ Content-Type: application/json
 -   **200** - no error
 -   **404** - plugin not installed
 
-<!-- TODO Document "docker plugin set" endpoint once implemented
 ### Configure a plugin
 
-`POST /plugins/(plugin name)/set`
+POST /plugins/(plugin name)/set`
 
-**Status codes**:
+**Example request**:
 
--   **500** - not implemented
 
--->
+    POST /plugins/tiborvass/no-remove/set
+    Content-Type: application/json
+
+    ["DEBUG=1"]
+
+**Example response**:
+
+    HTTP/1.1 204 No Content
+
+**Status codes**:
+
+-   **204** - no error
```

Comment:
`StatusOK` is supposed to be accompanied with a body, set returns none so `StatusNoContent`, it's the convention we followed for other endpoints.